docs/devguide/translation.rst

```diff
@@ -0,0 +1,43 @@
+.. _dev-translate:
+
+============
+Translations
+============
+
+The Python community is a diverse and cosmopolitan place. Our documentation
+should be available for users in many different languages and for many different
+locales.
+
+This document explains how to go about translating this documentation.
+
+Read The Docs
+-------------
+
+Currently we host our documentation on http://readthedocs.org. We use Sphinx to
+make the documentation happen.
+
+The relevant documentation for localisation in ReadTheDocs is here:
+
+https://docs.readthedocs.io/en/latest/localization.html#project-with-multiple-translations
+
+Put simply, each new translation is a separate project, presumably a Sphinx
+based repository on GitHub, that contains the translated resources and is linked
+to ReadTheDocs in the usual way - making sure its locale is set  appropriately.
+
+It's possible for this "core" project to link to these other projects as
+"translations of" this one.
```

Comment:
Right, so the administrators would basically add the repository links for each supported language. Is that configuration page also used to indicate which language the repository represents? Or does the repository creator need to do anything to indicate the language?